Lib/test/test_email/test__header_value_parser.py

```diff
@@ -3255,5 +3255,18 @@ def test_long_filename_attachment(self):
             " filename*1*=_TEST_TES.txt\n",
             )
 
+    def test_fold_unfoldable_element_stealing_whitespace(self):
+        # gh-142006: When an element is too long to fit on the current line
+        # the previous line's trailing whitespace
+        policy = self.policy.clone(max_line_length=10)
+
+        # "a," fits. The space after it should wrap to the next line.
+        # The "long" part (20 chars) forces the wrap because 20 > 10.
+        text = "a, " + ("b" * 20)
+        expected = "a,\n " + ("b" * 20) + "\n"
+
```

Comment:
I'd drop the blank lines.  They don't add anything and most other tests don't have internal blank lines.